Lab03-WordGuessGame/Program.cs

```diff
@@ -36,14 +36,14 @@ static public int UserInterfaceMenu()
             try
             {
                 Console.WriteLine("Hello contestant!, Welcome to Word Guess Game.");
+                Console.WriteLine();
                 Console.WriteLine("1: New Game");
                 Console.WriteLine("2: Add A Word");
                 Console.WriteLine("3: View Words");
                 Console.WriteLine("4: Delete Words");
```

Comment:
You don't want to delete all words, you want to delete a single word form the file and leave the rest alone